.github/workflow/lint.yml

```diff
@@ -0,0 +1,37 @@
+name: Lint Charts
+
+on: pull_request
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.11.2
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+          check-latest: true
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.4.0
```

Comment:
Looking at https://github.com/helm/chart-testing-action gives me no indication that this will work without a `charts` directory. Happy to be wrong

Reply:
I added CT env var just in case